src/main/java/com/rarchives/ripme/ripper/AbstractRipper.java

```diff
@@ -563,6 +563,10 @@ public void run() {
             LOGGER.error("Got exception while running ripper:", e);
             waitForThreads();
             sendUpdate(STATUS.RIP_ERRORED, "HTTP status code " + e.getStatusCode() + " for URL " + e.getUrl());
+        }catch(NullPointerException e){
+            LOGGER.error("Got null pointer exception while running ripper:", e);
+            waitForThreads();
+            sendUpdate(STATUS.NO_ALBUM_OR_USER, "Album or user doesn't exist!");
```

Comment:
You should make sure that the status code is 404 before throwing the error message, it's possible for a NullPointerException to the thrown for reasons besides the url not existing 

Also `}catch(NullPointerException e){` should be `} catch(NullPointerException e) {` for style reasons